src/modules/AccordionGrommet.js

```diff
@@ -0,0 +1,63 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from 'grommet/components/Box';
+import Headline from 'grommet/components/Headline';
+import List from 'grommet/components/List';
+import Paragraph from 'grommet/components/Paragraph';
+
+const CLASS_ROOT = 'accordion-grommet';
+
+export default class AccordionGrommet extends Component {
+  render () {
+    let { headline, subHeadline, children } = this.props;
+
+    let classes = classnames(
+      CLASS_ROOT,
+      this.props.className
+    );
+
+    let headlineMarkup;
+    if (headline) {
```

Comment:
Just as good practice, can we break this function out into multiple smaller ones that each does a single thing?